examples/backend/src/ExampleAPI.hs

```diff
@@ -39,8 +52,27 @@ data User = User
   deriving (Generic)
   deriving (Elm, ToJSON, FromJSON) via ElmStreet User
 
+data SortBy = Age | Name
+  deriving (Show, Generic, Elm)
+
+instance ToJSON SortBy
```

Comment:
Use the `DerivingVia` style for `Elm` and `ToJSON`

Reply:
done